### ISSUE TITLE
Feature request: view values for lists and dicts

### ISSUE THREAD

**Issue by sam-m-caldwell** [yhat/rodeo#478]
Really enjoying Rodeo so far and hoping it will get me out of Spyder soon! As is possible with Spyder, would really love to be able to click in to view the values within lists and dicts, similar to what is available already for dataframes.

**Comment by TakenPilot** [comment]
Agreed!  I'm working on a TreeView component for the files, and I want to use the same component to show more information about the current state of the environment as well.
